bundles/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/SqueezeBoxAudioSink.java

```diff
@@ -88,8 +88,8 @@ public void processSynchronously(AudioStream audioStream)
         }
 
         String url;
-        if (audioStream instanceof URLAudioStream) {
-            url = ((URLAudioStream) audioStream).getURL();
+        if (audioStream instanceof URLAudioStream stream) {
+            url = stream.getURL();
```

Comment:
More specialized rather than less:
```suggestion
        if (audioStream instanceof URLAudioStream urlAudioStream) {
            url = urlAudioStream.getURL();
```